monkey/infection_monkey/utils/linux/shell_startup_files_modification.py

```diff
@@ -0,0 +1,13 @@
+BASH_STARTUP_FILES = ["~/.bashrc", "~/.profile", "~/.bash_profile"]
+
+
+def get_linux_commands_to_modify_shell_startup_files():
+    return [
+        'echo \"# Succesfully modified {0}\"',
+        '3<{0} 3<&- |',  # check for existence of file
+        'tee -a',  # append to file
+        '{0}',
+        '&&',
+        'sed -i \'$d\' {0}',  # remove last line of file
+    ],\
+    BASH_STARTUP_FILES
```

Comment:
This will work in case the user is using `bash`, but if the default shell is different (`zsh` etc.)

Reply:
The command will work but we'll have to add the startup file names for the other shells to `BASH_STARTUP_FILES`, then

Reply:
@ShayNehmad Any reason to actually support all the shells? 
sh, bash, dash. Those are standard?

Reply:
@acepace If it's not difficult (adding paths of `zsh`, `fish`, `dash`, `sh`... `ksh` and `tcsh` maybe) then yeah, but if it requires a lot of research and testing (and not only adding paths to the STARTUP_FILES const) then it's not worth it. 
@shreyamalviya your call :)

Reply:
@ShayNehmad @acepace it just requires adding the paths to the STARTUP_FILES constant but would it be a good idea to include other shells, given that the name of the technique is _.bashrc and .bashprofile_?

Reply:
I like your last sentence.